k8s/ithc/cluster-01-overlay/rd/kustomization.yaml

```diff
@@ -4,6 +4,8 @@ namespace: rd
 bases:
 - ../../../namespaces/rd/rd-judicial-data-load/rd-judicial-data-load.yaml
 - ../../../namespaces/rd/rd-location-ref-data-load/rd-location-ref-data-load.yaml
+- ../../../namespaces/rd/rd-commondata-dataload/rd-commondata-dataload.yaml
 patchesStrategicMerge:
 - ../../../namespaces/rd/rd-judicial-data-load/ithc-01.yaml
-- ../../../namespaces/rd/rd-location-ref-data-load/ithc-00.yaml
\ No newline at end of file
+- ../../../namespaces/rd/rd-location-ref-data-load/ithc-00.yaml
```

Comment:
I know this is not as part of your task but can you change this to ithc-01.yaml?